eos/saveddata/fit.py

```diff
@@ -392,6 +399,9 @@ def clear(self, projected=False):
         self.ecmProjectedStr = 1
         self.commandBonuses = {}
 
+        for remoterep_type in ("Armor", "Shield", "Hull", "Capacitor"):
+            self.__remoteReps[remoterep_type] = 0
+
```

Comment:
This is not that big of a deal since it's all hardcoded, but didn't know if you knew about this syntax (and if you use it we wouldn't have to update it here if the original ever changes)